### PR TITLE
release yaml for aip 57

### PR DESCRIPTION
**Pull request by igor-aptos**

### CODE REVIEW

**Comment by igor-aptos** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #11364** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-core/11364?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/aptos-labs/aptos-core/11364?utm_source=stack-comment).

**Comment by msmouse** [comment]
I don't think we check in the resulting artifacts? (also the folder name is..)

File: types/src/on_chain_config/execution_config.rs [hunk 1]
```diff
@@ -152,6 +152,7 @@ pub enum TransactionDeduperType {
 }
 
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+#[serde(rename_all = "snake_case")] // cannot use tag = "type" as nested enums cannot work, and bcs doesn't support it
```
Comment (msmouse) [inline types/src/on_chain_config/execution_config.rs:155]: hmm... as long as compat test passes, I guess..

**Review by msmouse — APPROVED** [review]

File: types/src/on_chain_config/execution_config.rs (hunk 1 above)
Comment (igor-aptos) [reply types/src/on_chain_config/execution_config.rs:155]: This is new for this release. 

But also,  names are only relevant for release tooling, not onchain

**Review by thepomeranian — APPROVED** [review]

**Pull request merged**